storage/database/db_manager_test.go

```diff
@@ -387,6 +387,49 @@ func TestDBManager_TrieNode(t *testing.T) {
 
 		hasStateTrieNode, _ = dbm.HasStateTrieNode(hash1[:])
 		assert.True(t, hasStateTrieNode)
+
+		if !dbm.IsPartitioned() {
+			continue
+		}
+		dbm.SetStateTrieMigrationDB(123)
+
+		cachedNode, _ = dbm.ReadCachedTrieNode(hash1)
```

Comment:
It's not directly related to this PR, but why use both array or slice as a function parameter? I think that slice can be used for both cases.

Reply:
Right. Just used the same function signature from go-ethereum. I will change this in a separated PR.